### PR TITLE
feat: Learning Hub - Milestone 2

### PR DESCRIPTION
**Pull request by camielvs**
## Description

Removes the unnecessary fat from the guided tours mockup and turns it into a user-ready experience.

Removes:

- uncreated tours
- "popular" and "new" tags
- pointless hover state on cards

Plus update tour durations

## Related Issue and Pull requests

Closes https://github.com/Shopify/oasis-frontend/issues/584

Closes https://github.com/Shopify/oasis-frontend/issues/621

Closes https://github.com/Shopify/oasis-frontend/issues/583

## Type of Change

- [x] New feature

## Checklist

- [ ] I have tested this does not break current pipelines / runs functionality
- [ ] I have tested the changes on staging

## Test Instructions

- Confirm that all four new guided tours work as expected, including interactions, tinstructions and other features, and have minimal edge cases
- Confirm that Tours are correctly marked as completed
- Confirm that the tours page and UI is complete
- Confirm that the existing V2 Editor and Pipelines continues to work as normal when not inside a tour

### CODE REVIEW

**Comment by camielvs** [comment]
* **#2491** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2491?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2435** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2435?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2440** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2440?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2421** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2421?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2416** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2416?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2416?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#2417** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2417?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2415** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2415?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2412** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2412?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2406** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2406?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2405** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2405?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2366** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2366?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2365** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2365?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2312** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2312?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2347** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2347?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2306** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2306?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#2307](https://github.com/TangleML/tangle-ui/pull/2307) <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2307?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#2299** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2299?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2339** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2339?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2334** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2334?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2340** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2340?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2348** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2348?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2297** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2297?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2341** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2341?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2292** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2292?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2291** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2291?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2283** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2283?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2282** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2282?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2281** <a href="https://app.graphite.com/github/pr/TangleML/tangle-ui/2281?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: src/components/Learn/TourCompletedBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Badge } from "@/components/ui/badge";
+import { Icon } from "@/components/ui/icon";
+
+export function TourCompletedBadge() {
+  return (
+    <Badge size="sm" variant="outline" className="text-green-600">
```
Comment (maxy-shpfy) [inline src/components/Learn/TourCompletedBadge.tsx:6]: > 🤖 This is an AI-generated code review comment.

The extracted completed-state badge centralizes the repeated markup, which is good, but the new component still passes a Tailwind color class directly into the Badge primitive. The Tangle UI primitive rule says not to style primitives through className; color intent should be represented by a semantic prop/variant or a Layer-3 pattern so visual language stays centralized.

Suggestion: Add/use a semantic completed/success badge variant or a small Layer-3 CompletedBadge pattern that owns the success styling without callers passing className to the primitive.

Rule: tangle-ui-review checklist §6 / project UI primitives rule: never pass className to a Tangle UI primitive; use semantic props or Layer-3 patterns.

**Review by maxy-shpfy — APPROVED** [review]

File: src/components/Learn/TourCompletedBadge.tsx (hunk 1 above)
Comment (camielvs) [reply src/components/Learn/TourCompletedBadge.tsx:6]: deferred to separate/future prs

**Comment by camielvs** [comment]
### Merge activity

* **Jul 7, 5:57 PM UTC**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.com/github/pr/TangleML/tangle-ui/2416).
* **Jul 7, 7:16 PM UTC**: [Graphite](https://app.graphite.com/github/pr/TangleML/tangle-ui/2416) rebased this pull request as part of a merge.
* **Jul 7, 7:19 PM UTC**: @camielvs merged this pull request with [Graphite](https://app.graphite.com/github/pr/TangleML/tangle-ui/2416).
